src/tools/clippy/clippy_lints/src/transmute/mod.rs

```diff
@@ -271,7 +271,7 @@ declare_clippy_lint! {
     /// Use instead:
     /// ```no_run
     /// # use core::num::NonZeroU32;
-    /// let _non_zero = unsafe { NonZeroU32::new_unchecked(123) };
+    /// let _non_zero = unsafe { NonZero::new_unchecked(123) };
```

Comment:
I think this line should not be changing, as this is user-facing documentation that gets rendered to <https://rust-lang.github.io/rust-clippy/master/index.html#/transmute_int_to_non>. We can update this after stabilization together with updating the docs in library/core/src/num/nonzero.rs.